src/main/java/org/dplava/git/GithubValidityRegistry.java

```diff
@@ -91,29 +94,51 @@ public String getCommitStatus(URI repo, String commitHash) throws IOException {
     }
 
     @Override
-    public void reportCommitInvalid(URI repo, String commitHash, String url) throws IOException {
-        postStatus(repo, commitHash, FAILURE, url);
+    public void reportCommitInvalid(GithubPayload payload, String url) throws IOException {
+        postStatus(payload, FAILURE, url);
+        
+        //notify pusher via email
+        try {
+            Email email = new SimpleEmail();
+            email.setHostName("localhost");
+            email.setSmtpPort(465);
+            email.setFrom("user@gmail.com");
```

Comment:
This e-mail address should be passed in by a variable.

I'm curious, did you test this?  Did you receive an e-mail?